core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -670,10 +677,14 @@ class SparkContext(config: SparkConf) extends Logging {
    * sc.cancelJobGroup("some_job_to_cancel")
    * }}}
    *
-   * If interruptOnCancel is set to true for the job group, then job cancellation will result
-   * in Thread.interrupt() being called on the job's executor threads. This is useful to help ensure
-   * that the tasks are actually stopped in a timely manner, but is off by default due to HDFS-1208,
-   * where HDFS may respond to Thread.interrupt() by marking nodes as dead.
+   * @param groupId job group's identification
+   * @param description job group's decription
```

Comment:
The `interruptOnCancel` doc makes sense but these don't say much beyond the name. I'd expand them or remove them.